source/common/http/http1/codec_impl.cc

```diff
@@ -49,6 +49,12 @@ struct Http1ResponseCodeDetailValues {
   const absl::string_view ChunkedContentLength = "http1.content_length_and_chunked_not_allowed";
   const absl::string_view HttpsInPlaintext = "http1.https_url_on_plaintext_connection";
   const absl::string_view InvalidScheme = "http1.invalid_scheme";
+  const absl::string_view InvalidHost = "http1.invalid_host";
+  const absl::string_view InvalidPath = "http1.invalid_path";
+  const absl::string_view InvalidMethod = "http1.invalid_method";
+  const absl::string_view InvalidStatus = "http1.invalid_status";
+  const absl::string_view InvalidHeader = "http1.invalid_header";
+  const absl::string_view InvalidHeaders = "http1.invalid_headers";
```

Comment:
is there value in disambiguating 1 from 1+?

Reply:
Removed after refactor